.travis/klee.sh

```diff
@@ -208,6 +208,20 @@ else
 
   set +e # We want to let all the tests run before we exit
   lit -v .
+
+  # If metaSMT is the only solver, then rerun tests with non-default metaSMT backends
+  if [ "X${SOLVERS}" == "XmetaSMT" ]; then
+    if [ "X${METASMT_DEFAULT}" != "Xbtor" ]; then
+      lit -v --param klee_opts=-metasmt-backend=btor --param kleaver_opts=-metasmt-backend=btor .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xstp" ]; then  
+      lit -v --param klee_opts=-metasmt-backend=stp --param kleaver_opts=-metasmt-backend=stp .
+    fi
+    if [ "X${METASMT_DEFAULT}" != "Xz3" ]; then
+      lit -v --param klee_opts=-metasmt-backend=z3 --param kleaver_opts=-metasmt-backend=z3 .
+    fi
+  fi
+
```

Comment:
This part is also should be run during a CMake build too so it should pulled out of the `else` branch and appropriately modified to work for the CMake build (just make the path `test/` rather than `.`).